src/Common/Core/Impl/Shell/PDFExportOptions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Common.Core {
+    public class PDFExportOptions {
```

Comment:
Similar to `PdfPrintOptionsEnum`, should be in the same namespace and assembly with either dialog or command.